packages/zowe-explorer/i18n/sample/src/utils/ProfileManagement.i18n.json

```
@@ -22,6 +22,8 @@
   "editProfileQpItem.editProfile.qpDetail": "Update profile connection information",
   "hideProfileQpItems.hideProfile.qpLabel": "$(eye-closed) Hide Profile",
   "hideProfileQpItems.hideProfile.qpDetail": "Hide profile name from tree view",
+  "switchAuthenticationQpItems.switchAuthentication.qpLabel": "$(key) Change the Authentication Method",
+  "switchAuthenticationQpItems.switchAuthentication.qpDetail": "To change the authentication method",
```

I think we can remove "To" here since the other entries in this menu start with actions:

```suggestion
  "switchAuthenticationQpItems.switchAuthentication.qpDetail": "Change the authentication method",
```